Library/Formula/folly.rb

```diff
@@ -0,0 +1,47 @@
+class Folly < Formula
+  desc "An open-source C++ library developed and used at Facebook."
+  homepage "https://github.com/facebook/folly"
+  url "https://github.com/facebook/folly/archive/v0.47.0.tar.gz"
+  sha256 "7d3309088cae1817d60996611e5fc0825c5c8df36d244f247c1d49ebc38c35fb"
+  version "0.47.0"
```

Comment:
Can remove the explicit version now that you're using a tagged release; it can be inferred from the tarball name.